spec/models/ability/form_ability_spec.rb

```diff
@@ -24,13 +24,25 @@
   context "mission mode" do
     let(:ability) { Ability.new(user: user, mode: "mission", mission: get_mission) }
 
+    context "for admin" do
+      let(:user) { create(:user, admin: true) }
+
+      it "should allow re-caching forms" do
+        expect(ability).to be_able_to(:re_cache, Form)
+      end
+    end
+
     context "for coordinator" do
```

Comment:
do we need to add another test for other roles to make sure they can't re-cache? Or is this good enough?

Reply:
The below test for enumerators should cover everything else (since only coordinators and admins are touched in abilities).